README.md

```diff
@@ -43,10 +41,12 @@ Then setup these servers, and other defaults in your environment.rb:
 ```rb
 GlobalUid::Base.global_uid_options = {
   id_servers: [ 'id_server_1', 'id_server_2' ],
-  increment_by: 3
+  increment_by: 5
 }
 ```
 
+The `increment_by` value configured here does not dictate the value on your alloc server and must remain in sync with the value of [`auto_increment_increment`](https://dev.mysql.com/doc/refman/5.6/en/replication-options-master.html#sysvar_auto_increment_increment)
```

Comment:
This disclaimer and the references below it will be updated once the validation work in #63 is merged